### ISSUE TITLE
Dapr

### ISSUE THREAD

**Issue by juntao** [second-state/chat-with-chatgpt#15]
Do you know the Dapr project?

**Comment by juntao** [comment]
Hello

**Comment by juntao** [comment]
Do you know the Dapr project?

**Comment by juntao** [comment]
Is it a service mesh?

**Comment by juntao** [comment]
Can I create a Wasm app in Dapr?

**Comment by juntao** [comment]
Is there a Dapr SDK that allows a Wasm app to communicate with a Dapr sidecar?

**Comment by juntao** [comment]
Can you summarize the README page for the Dapr WASM SDK?

**Comment by juntao** [comment]
Great! You mentioned that Dapr supports many programming languages. Which languages does the Dapr WASM SDK support?

**Comment by juntao** [comment]
Well, if you actually look into the GitHub repo, you will find that the Dapr WASM SDK currently only supports Rust.

**Comment by juntao** [comment]
This is incorrect. The Dapr WASM SDK requires special WASI socket networking enhancements that are only available in the WasmEdge runtime.

**Comment by juntao** [comment]
So, I will ask again. Does the Dapr WASM SDK require a specific Wasm runtime to work?

**Comment by juntao** [comment]
Thank you.

**Comment by benjaminhuo** [comment]
What's the current status of Dapr WASM SDK?
